src/components/GlobalFooter.js

```diff
@@ -42,7 +45,7 @@ const FOOTER_LINKS = [
 
 export default () => (
   <footer className="GlobalFooter">
-    <div className="container text-center">
+    <div className="text-center footer-left">
       <span className="list-inline-item mb-2">ChicagoJS {new Date().getFullYear()}</span>
       <ul className="list-unstyled list-inline mp-0">
         {FOOTER_LINKS.map(link => (
```

Comment:
While you're in here, can you add a className called "FooterSocialLink" with the styles of  `cursor: pointer`. 
That way the user knows it's more clickable

Reply:
Can you elaborate a bit? The social media icons already use the ```cursor:pointer``` style, so I'm a bit confused as to which element should get that class name?

Reply:
My b, just noticed that. The slack one didn't have a URL so it wasn't working on that one element.